src/scss/grommet-core/_objects.video.scss

```diff
@@ -37,6 +37,10 @@
 
   video {
     width: 100%;
+
+    .video--poster {
+      width: 100vw;
```

Comment:
The poster defaults to the class `.video video { width: 100%; } `. `vw` would just be a more browser consistent solution.